Tools/Gulp/gulp-addModuleExports.js

```diff
@@ -5,7 +5,7 @@ module.exports = function (varName) {
   return through.obj(function (file, enc, cb) {
 
     var moduleExportsAddition = 
-      '\nif (((window && window.module) || (!window && module)) && module.exports) {\n' +
+      '\nif (((window && window.module) || (typeof window != "undefined" && typeof module != "undefined")) && typeof module.exports != "undefined") {\n' +
```

Comment:
Can you try with:
if (((typeof window !== "undefined" && window.module) || (typeof module != "undefined")) && typeof module.exports != "undefined")